relayer/pkg/starknet/deployer_starknet.go

```diff
@@ -1,4 +1,4 @@
-package integration_tests
+package starknet
```

Comment:
Why this move?

The deployer should be moved back to `integration_tests` or removed completely if not used anymore, as I think we have a new Gauntlet deployer now.

Reply:
Removed since unused